test/functional/helpers/spectrum.js

```diff
@@ -253,12 +255,12 @@ Object.keys(componentWrappers).forEach(componentName => {
   ) {
     const selector = selectorize(testIdOrSelector);
     return {
+      expectEnabled: createExpectEnabled(selector),
+      expectDisabled: createExpectDisabled(selector),
       ...componentWrapper.call(this, selector),
       selector,
       expectExists: createExpectExists(selector),
-      expectNotExists: createExpectNotExists(selector),
-      expectEnabled: createExpectEnabled(selector),
-      expectDisabled: createExpectDisabled(selector)
+      expectNotExists: createExpectNotExists(selector)
```

Comment:
I like what you did here. I'd recommend moving `expectExists` and `expectNotExists` up as well so the individual component wrappers could override that if they wanted to, however unlikely.